tethysext/atcore/urls/resources.py

```diff
@@ -0,0 +1,169 @@
+"""
+********************************************************************************
+* Name: resources.py
+* Author: msouffront & htran
+* Created On: November 20, 2020
+* Copyright: (c) Aquaveo 2020
+********************************************************************************
+"""
+import inspect
+from django.utils.text import slugify
+from tethys_sdk.base import TethysController
+from tethysext.atcore.controllers.app_users import ManageResources, ModifyResource, ResourceDetails, ResourceStatus
+from tethysext.atcore.models.app_users import AppUser, Organization, Resource
+from tethysext.atcore.services.app_users.permissions_manager import AppPermissionsManager
+
+
+def urls(url_map_maker, app, persistent_store_name, base_url_path='', base_template='atcore/app_users/base.html',
```

Comment:
Please be sure to test this function on it's own (i.e.: not just through testing the `app_users.urls` function). In fact, this function should be mocked out in the `app_users.urls` function tests to help separate concerns.